node/Cargo.toml

```diff
@@ -39,6 +39,7 @@ jsonrpc-core = "15.1.0"
 # Substrate frames
 frame-benchmarking = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
 frame-benchmarking-cli = { version = '3.0.0', git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.8" }
+try-runtime-cli = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.8", optional = true }
```

Comment:
may need to specify a version here.

Reply:
You got it. Included the version explicitly it is 0.9.0, but don't get it confused with the polkadot v0.9.8